contracts/BC_fusion/StakeHub.sol

```diff
@@ -137,6 +148,14 @@ contract StakeHub is System, Initializable {
     mapping(address => bool) public blackList;
 
     /*----------------- structs and events -----------------*/
+    struct MigrationPackage {
+        address operatorAddress;
+        address delegator;
+        address refundAddress;
+        uint256 amount;
```

Comment:
Please add comments for all fields. Also the decimal of the amount here.

Reply:
Done